input/pagecontent/conformance.md

```diff
@@ -1,10 +1,135 @@
+- [FHIR RESTful API](http://hl7.org/fhir/R4/http.html) 
+- [FHIR Search](http://hl7.org/fhir/R4/search.html)
+- [FHIR Resource formats](http://hl7.org/fhir/R4/resource.html)
+- [FHIR Data Types](http://hl7.org/fhir/R4/datatypes.html)
+- [FHIR Workflow](https://hl7.org/fhir/R5/workflow.html)
 
+Additionally, implementers are advised to be familiar with the AU Core requirements on:
+- [Missing Data](https://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html#missing-data)
+- [Suppressed Data](https://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html#suppressed-data)
 
+The requirements of the FHIR standard and [FHIR Conformance Rules](http://hl7.org/fhir/conformance-rules.html) apply, and define the use of terms in this guide including the conformance verbs - **SHALL**, **SHALL NOT**, **SHOULD**, **SHOULD NOT**, **MAY**.
 
+### Conformance Artifacts
 
+#### AU eRequesting Profiles and Extensions
+The [Profiles and Extensions](profiles-and-extensions.html) page lists the AU eRequesting profiles and AU eRequesting extensions defined for this implementation guide. An AU eRequesting profile [StructureDefinition](http://hl7.org/fhir/R4/structuredefinition.html) defines the minimum elements, extensions, vocabularies and value sets that **SHALL** be present and constrains the way elements are used when conforming to the profile.
 
+AU eRequesting profile elements include mandatory and *Must Support* requirements. [Mandatory elements](#mandatory-elements) are required and have a minimum cardinality of 1 (min=1). [Must Support](#must-support-and-obligation) elements have defined conformance obligations in AU eRequesting based on actor roles.
+
+#### AU eRequesting Capability Statements
+The [AU eRequesting Placer CapabilityStatement](CapabilityStatement-au-erequesting-placer.html) defines the conformance requirements and expectations of an [AU eRequesting Placer](ActorDefinition-au-erequesting-actor-placer.html) actor responsible for initiating and creating diagnostic service requests. The complete list of FHIR profiles, REST API interactions, and search parameters that can be implemented by an AU eRequesting Placer are defined in this capability statement. AU eRequesting placers define their capabilities by choosing from this list based on the resource types they need need to place requests. 
+
+The [AU eRequesting Server CapabilityStatement](CapabilityStatement-au-erequesting-server.html) defines the conformance requirements and expectations of an [AU eRequesting Server](ActorDefinition-au-erequesting-actor-server.html) actor responsible for accepts diagnostic service requests and makes diagnostic service requests accessible. The complete list of FHIR profiles, REST API interactions, and search parameters that can be implemented by an AU eRequesting Server are defined in this capability statement. An AU eRequesting Server declares conformance to this list of capabilities based on the resource types and interactions it implement.
```

Comment:
"..responsible for accepts diagnostic service requests and makes diagnostic service requests accessible..."
change to 'accepting', 'making'

Reply:
'implement', change to 'implements'